### PR TITLE
Task7 Abstract-Interfaces

### PR DESCRIPTION
**Pull request by polinaK4**

### CODE REVIEW

File: InterfacesTask/ClassDuck.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                Console.WriteLine($"Step...");
+            }
+        }
+        public Duck(string description, int Steps, int Height)
+        {
+            this.description = description;
+            this.Steps = Steps;
+            this.Height = Height;
+        }
+
```
[the remark this answers is not in the document]
Comment (polinaK4) [reply InterfacesTask/ClassDuck.cs:33]: fixed

File: ElectronicStore/Printer.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+{
+    public class Printer : Device, IPrint
+    {
+        public int PaperWidth { get; set; }
+        public int PaperHeight { get; set; }
+
+        public Printer(string? modelName, decimal price, int PaperWidth, int PaperHeight) : base(modelName, price)
+        {
+            this.PaperWidth = PaperWidth;
+            this.PaperHeight = PaperHeight;        }
```
[the remark this answers is not in the document]
Comment (polinaK4) [reply ElectronicStore/Printer.cs:11]: fixed

File: ElectronicStore/AbstractClassDevice.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    {
+        public string? modelName;  
+        public decimal price; 
+        public abstract string Description { get; }
+        public abstract void TurnOn(); 
+        public abstract void TunrnOff();
+        public Device(string modelName, decimal price)
+        {
+            this.modelName = modelName;
+            this.modelName = modelName;
```
[the remark this answers is not in the document]
Comment (polinaK4) [reply ElectronicStore/AbstractClassDevice.cs:13]: done

File: AbstrInterfaces/AbstractClassTransport.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
+﻿namespace AbstractClasses
+{
+    abstract class Transport
+    {
+        public string name = "Transport";
+        public abstract int Speed { get; set; }
+        public abstract void Move();
+        public void PrintTheName()
```
[the remark this answers is not in the document]
Comment (polinaK4) [reply AbstrInterfaces/AbstractClassTransport.cs:8]: done

**Review by pickslayer17 — APPROVED** [review]

**Pull request merged**